test_e2e.sh

```diff
@@ -13,5 +13,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-./test/e2e/ci.sh
+REF="${{ github.base_ref }}"
+CHECK_DOCS_ONLY=$("test/check-docs-only.sh $REF")
```

Comment:
@camilamacedo86 Are the changes needed, related to $REF?

Reply:
It is not working yet
```
Done setting up docker in docker.
+ WRAPPED_COMMAND_PID=164
+ ./test_e2e.sh
+ wait 164
./test_e2e.sh: line 16: ${{ github.base_ref }}: bad substitution
./test_e2e.sh: line 1[7](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kubebuilder/2687/pull-kubebuilder-e2e-k8s-1-14-10/1527237003974807552#1:build-log.txt%3A7): test/check-docs-only.sh : No such file or directory
```

Reply:
The prow will not be able to check GitHub events: `REF="${{ github.event.ref }}"`